test/units/plugins/inventory/test_inventory.py

```diff
@@ -173,7 +173,7 @@ def test_yaml_inventory(self):
         """)}
         C.INVENTORY_ENABLED = ['yaml']
```

Comment:
@pilou- C.INVENTORY_ENABLED must be reset after using?

Reply:
Yes. The new method is inside `IniInventory` class, it could be moved in another class (`YamlInventory` ?), then `setUp` and `tearDown` methods could be used in order to 1. save, 2. set (in `setUp`) and 3. restore (in `tearDown`) `C.INVENTORY_ENABLED`.